docs/Gemfile.lock

```diff
@@ -6,12 +6,12 @@ GEM
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (10.4.2.0)
+    addressable (2.8.1)
```

Comment:
as you mentioned yesterday this was the issue, I've updated this and merged to main so hopefully if you rebase this is no longer needed

Reply:
Should we still run `bundle update` tho?

Reply:
It worked for me with just making sure the ruby version was correct